### DISCUSSION TITLE
detectron2 whl distribution rule

### DISCUSSION THREAD

**Discussion by ashnair1** [facebookresearch/detectron2#2836] [Q&A]
Hi,

Is there any particular reason there is no detectron2 whl for torch 1.8, CUDA 11?

While on this subject, how is the whl distribution done? For example  for CUDA 11.0, the only available  detectron2 whl is for torch 1.7 whereas for CUDA 10.1 and 10.2 there are whls for all torch versions (1.6 to 1.8). I would like to know how whl distribution is dependent on the pytorch and CUDA versions

**Comment by ppwwyyxx** [comment]
> Is there any particular reason there is no detectron2 whl for torch 1.8, CUDA 11?

because there is no pytorch whl for this combination.

>  I would like to know how whl distribution is dependent on the pytorch and CUDA versions

we follow what pytorch wheels are released in https://download.pytorch.org/whl/torch_stable.html

**Reply by ashnair1** [reply]
Ah I see. Would you happen to know why pytorch doesn't release those whls? Is it due to some underlying incompatibility?

**Reply by ppwwyyxx** [reply]
I'm not sure why

**Reply by Maciej-Zwolinski** [reply]
@ppwwyyxx 

Where can I find these wheels? I'm looking for torch==1.10.2+cu113 (which is available at the resource you pointed to in the original response), however trying different variations of `https://dl.fbaipublicfiles.com/detectron2/wheels/cu113/torch1.10.2/detectron2-0.3%2Bcu113-cp38-cp38-linux_x86_64.whl` gives no results.

**Comment by MiroPsota** [comment]
You can try my repository for building packages and PyPI simple index and see if it works for you:
https://github.com/facebookresearch/detectron2/discussions/5200
